manifesttool/mtool/actions/create.py

```diff
@@ -91,7 +91,7 @@ def register_parser_args(
             required.add_argument(
                 '-v', '--fw-version',
                 type=semantic_version_arg_factory,
-                help='Version number of the candidate image in SemVer format.',
+                help='Version number of the candidate image in SemVer format. Minimum 0.0.2, maximum 999.999.999.',
```

Comment:
@JanneKiiskila , Why the minimum is `0.0.2`? Currently, the tool allows `.0.0.0`.
```suggestion
                help='Version number of the candidate image in SemVer format. Maximum 999.999.999.',
```

Reply:
What is the base version the original image can have? Is it 0.0.0 and you can flash 0.0.0 on top of it, or is it 0.0.1?

Reply:
I.e. what version number enforcement do we have? Should this parameter also be mandatory? @moshe-shahar ?

Reply:
Yes, the base SemVer is `0.0.0`.
Currently, the tool and the client limits are `0.0.0`-`999.999.999`.
Also, the client will reject any manifest with less or equal to the current version.
So, de facto, you can not update the device to `0.0.0`.
We will consider limiting the minimum in the tool to `0.0.1`.

Reply:
> Should this parameter also be mandatory?

I expect it to be required param. if for some reason it's not, this is a bug.